src/libraries/wizard/selectsourcepage.cpp

```diff
@@ -56,7 +56,12 @@ SelectSourcePage::SelectSourcePage(ComponentImporter &importer, QWidget *parent)
             this,
             &SelectSourcePage::completeChanged);
 
-    m_ui.builtInRadio->setChecked(true);
+    if (configHasBuiltInLibraries()) {
+        m_ui.builtInRadio->setChecked(true);
+    } else {
+        m_ui.builtInRadio->setEnabled(false);
+        m_ui.customRadio->setChecked(true);
+    }
```

Comment:
I am not sure, if wizard pages are created every time when wizard is opened, or maybe only for the first time. In the latter case, this solution would prevent builtInRadio from ever being activated, after adding library.

Reply:
Code suggest that wizard can be reused, but quit test shows that it does not happen... hmm... just to be on the safe side I'll change it

Reply:
moved to initializePage